### DISCUSSION TITLE
Where is your light node?

### DISCUSSION THREAD

**Discussion by HUX13Y** [MyFirstBitcoin/.github#9] [LNN Introductions]
Where in the world are you operating your node and who are you teaching?

I am based in the U.K. in South East London. I have had two successful cohorts complete the MPB diploma and I am now teaching my third cohort. We meet weekly on Mondays between 1530 and 1700. My students are 12-17 yo from the school that I teach at. I am still refining my practise both in my day-job and as a fledgling Bitcoin-educator; there is so much to learn!

I am looking forward to learning more about the other node operators.

**Comment by CoachCarbon** [comment]
Bitcoin Ballers Academy, currently operating out of Oxford in the UK. Will be starting our enrolment soon with a start date of January 21st 2024.

Would like to attend one of your meets if you have one early in the New Year please.

**Reply by HUX13Y** [reply]
Hey! Yes absolutely we will set a meeting up in early Jan.

**Comment by mumo1mumo** [comment]
Hi everyone. My name is Mumo, a Bitcoin content creator based in Nairobi, Kenya. I am building SatoshifyMe education platform aimed at educating people about Bitcoin. The course I am curating is a self-paced online course where they can learn using the MPB diploma format, customised to our experience in Africa. The first version is being built in English and the second iteration will be in Swahili. Course launch is happening in March 2024.

**Reply by HUX13Y** [reply]
Hi Mumo! Thanks for sharing. I am very curious to see how your project progresses, especially because you are developing an online curriculum. We have one teacher in our node who will be developing an online curriculum for the students at his online school. How long does it typically take your students to finish the course?

**Comment by bertavalle** [comment]
Hola! Soy Berta. 
Mi principal interes es llevar mas conocimiento sobre libertad Financiera y Bitcoin a #Nicaragua, particularmente a los defensores de Derechos humanos y Activistas.

**Reply by HUX13Y** [reply]
¡Hola! Berta,
Su viaje para llevar más conocimiento sobre Bitcoin a los defensores de derechos humanos y a #Nicaragua es admirable!

**Comment by BitcoinVictoriaFalls** [comment]
Hi Everyone from Livingstone, Zambia!

**Comment by Podosiri** [comment]
Hello everyone. I'm planning on using English to teach the Bitcoin Diploma in Paramaribo, Suriname. I hope this is a suitable place to fulfill Step 2 in the Mi Primer Bitcoin application, but that's my intent :)

**Reply by HUX13Y** [reply]
Hi @Podosiri that sounds excellent, I am looking forward to following your progress!

**Comment by isabellasg3** [comment]
Hello everyone! It amazing to see everything you guys have done. 

I am Isabella Santos, I have been in the Bitcoin space since 2021 and have been working on educating others about Bitcoin since then. From the start, I have always wanted to give people the tools to learn and make a choice for themselves. In June of 2023, I decided to quit my fiat job and focus my 100% in the bitcoin community. I started a show called Bitcoin Backstage for Bitcoin Magazine in order to reach the masses and wake them up from the untold truths about money, the government, and everything else.

I went to Peru alongside another Bitcoin creator, Julian Figueroa, to document the impact MOTIV has been doing down there in the isolated parts of my home country. That was when I decided that I needed to do something more. So I packed my bags and moved to Isla Mujeres, a small island off the coast of Cancun, to start small and make a change in a community that needs it most.

I was inspired by the work that Valentin Popescu has done and wanted to create that in Mexico. Obviously, I know I cannot do it alone. That is why I need the help of Mi Primer Bitcoin to help me educate the people who reside in Isla Mujeres. I want BTC Isla to be a hub for freedom and Bitcoin.

I have spoken to the people in charge of making Roatan a Bitcoin-friendly island for some tips.

So far, I have managed to find a location to host meet-ups and a place to educate people as well. I also found a lightning wallet that is willing to help onboard businesses. I plan to divide my time by trying to get locals, businesses, after-school students, taxi drivers, foreigners who have moved here to retire, and other target groups.

But I can't do any of that without properly educating them about Bitcoin. I was thinking on having two different languages Spanish and English. But for the beginning, I think I will mainly use Spanish. I am bilingual, so I can teach in both.

So yeah, I would love to be a node in Mi Primer Bitcoin. I could use the support and guidance because I might be in over my head, but I know in the end it will be worth it.

**Comment by kestercrypt** [comment]
Hi everyone, I'm Kester Ejikeme, from Bitcoin Anambra Nigeria 🇳🇬.

**Reply by HUX13Y** [reply]
Welcome Kester, have you joined as a Light Node? How many students do you have in your class?

**Reply by kestercrypt** [reply]
I've applied as a light node but I'm yet to get a response.

I'm building a circular economy in my home state, and I plan on using the course to keep traders/merchants engaged online as well as giving them a platform to learn/understand Bitcoin better.

**Comment by Herrytheeagle** [comment]
Hello everyone, I am Heritage Falodun widely known as @herrytheeagle on all social platform. As part of broader plan of our company "DigiOats" in Nigeria to intensify Bitcoin adoption through education, we will be using MPB Diploma Curriculum and course content to educate everyone inn Nigeria, Africa and the Global south at large.

**Comment by jpgaviria2** [comment]
Hello, Hoping to be a light-node in Vancouver, Canada, Can provide education in English and Spanish. Recently supported learning bitcoin conference in Victoria Canada providing education on Bull Bitcoin workshops, met other MPB educators and got interested in being able to provide this education
